src/App.vue

```diff
@@ -4,20 +4,10 @@ import HelloWorld from './components/HelloWorld.vue'
 </script>
 
 <template>
-  <header>
-    <img alt="Vue logo" class="logo" src="@/assets/logo.svg" width="125" height="125" />
-
-    <div class="wrapper">
-      <HelloWorld msg="You did it!" />
-
-      <nav>
-        <RouterLink to="/">Home</RouterLink>
-        <RouterLink to="/about">About</RouterLink>
-      </nav>
-    </div>
-  </header>
-
-  <RouterView />
+  <div class="container mx-auto bg-gray-200 rounded-xl shadow border p-8 m-10">
```

Comment:
from this
![image](https://github.com/TheInfinityIQ/JustDucksClient/assets/45568848/39309449-c1a0-4ebb-a57f-4a906cbb2360)

this to
![image](https://github.com/TheInfinityIQ/JustDucksClient/assets/45568848/41433e54-19cb-4eb9-9a82-70604855c34d)

```suggestion
  <div class="container mx-auto rounded-xl shadow border p-8 m-10">
```